### ISSUE TITLE
Build instructions

### ISSUE THREAD

**Issue by MatteoBuffo** [davedoesdev/ffmpeg.js#5]
Hi! Are the Build with Docker instructions in README the ones we are expected to follow? I'm facing various errors while make-ing in the Docker image.

**Comment by vaishakhsnair** [comment]
yeah same here the instructions given in the README does'nt seem to work

**Comment by vaishakhsnair** [comment]
@MatteoBuffo  the build instruction is different for this one . Its given in a different issue which is now closed #3

**Comment by davedoesdev** [comment]
Yeah I need to update the build instructions. The problem is this is still a PR on upstream and if it's merged then the build instructions would revert to the original Docker image.

**Comment by davedoesdev** [comment]
These commands should build everything on Ubuntu:

```bash
sudo apt-get update
sudo apt-get install -y git python3 build-essential automake libtool pkg-config

cd ~
git clone https://github.com/emscripten-core/emsdk.git && cd emsdk
./emsdk install latest
./emsdk activate latest
source emsdk_env.sh

cd ~
git clone https://github.com/davedoesdev/ffmpeg.js.git --recurse-submodules && cd ffmpeg.js
make
```

Replace the last line with `make hls` or `make dash` to make only the HLS/DASH js and wasm files.
